presqt/api_v1/utilities/utils/function_router.py

```diff
@@ -1,5 +1,6 @@
 from presqt.osf.functions.fetch import osf_fetch_resources, osf_fetch_resource
 from presqt.osf.functions.download import osf_download_resource
+from presqt.osf.functions import upload
```

Comment:
Hey - is there a reason you wanted to have this third import styled differently?
Perhaps make the other two like this?